src/js/playlist/playlist.js

```diff
@@ -18,6 +18,8 @@ define([
         var providers = model.getProviders();
         var preload = model.get('preload');
         var feedid = model.get('feedid');
+        var feedInstanceId = model.get('feed_instance_id');
+        var pinSetId = model.get('pin_set_id');
```

Comment:
This solution works on setup, but after that, if another playlist is loaded, these model properties would be invalid and should not be added to each item.

@karimJWP @dannyfinks